src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -486,6 +486,9 @@ private Size ComputeLargeContentSize()
 
             var largeContentGridSize = largeContentGrid.DesiredSize;
 
+            // Add two times width of scroll bar (5) and right margin(5), refer to infoBubbleView.xaml
+            largeContentGridSize.Width += 20;
```

Comment:
@QilongTang , where are scroll bar and right margin defined? I can't find these magic numbers.